_dasldemos/proof-troubleshooting.md

```diff
@@ -0,0 +1,116 @@
+- Along the same lines, you can also look into the raw workflow log produced by your underlying PROOF server. To do this, identify your workflow log directory on the “PROOF Server” tab (see screenshot below) and navigate to that directory on Rhino. Once there, you will see a collection of files following the naming convention of `workflow.[workflowid].log`. Open the logfile corresponding to the workflow in question using your text viewer of choice (vi, vim, cat, VS Code) to see details about the Cromwell execution of your WDL script.
+
+![workflow_logs](/dasldemos/assets/proof_ts_workflow_logs.png)
+
+## Docker Container Issues
+
+### Creating your own Docker image
+
+- If you’re creating/troubleshooting your own Docker image and you push an update to the same version tag, you’ll need to clear your [Apptainer cache](https://apptainer.org/docs/user/main/cli/apptainer_cache_clean.html) for PROOF to download a new copy of the image. Otherwise, PROOF will just reuse the previous cached version of the image, therefore not incorporating your changes.
+- To remove the cached images, start by identifying your Apptainer cache directory at the bottom of the [“PROOF Server” tab](/dasldemos/proof-how-to#starting-a-proof-server) of PROOF (see screenshot below). Navigate to that directory in Rhino and delete all files in that directory ending in `.sif`.
+    - *Don’t delete the entire directory*. PROOF needs it to function properly and will not be able to properly submit jobs without it. If that does happen, you can recreate the directory by stopping and restarting your PROOF server.
+
+<img src="/dasldemos/assets/proof_ts_singularity_cache.png" alt="singularity_cache" width="600"/>
+
+### Hard link issues in Docker images
+
+- Users may be seeing issues related to linkages during the pull of Docker images that produce an error message along the lines of “operation not permitted":
+
+```FATAL:  While making image from oci registry: error fetching image to cache: while building SIF from layers: packer failed to pack: while unpacking tmpfs: error unpacking rootfs: unpack layer: unpack entry: usr/lib/python3.7/distutils/README: link: unpriv.link: unpriv.wrap target: operation not permitted```
+
+- Unfortunately, because the FH scratch directory doesn’t support hard links, Docker containers built with hard links cannot properly function on PROOF in the context of the Fred Hutch cluster system. Our developers are working on a [long-term solution](https://github.com/FredHutch/proof-api/issues/17) to this, but in the meantime, you’ll need to find/create an alternate version of the image that does not use hard links.
```

Comment:
The proof-api repo is private so people won't be able to see this.

Reply:
Ahhh, totally spaced about that, good catch. I've removed the link.